e2e/app/aou-elements/select-component.ts

```diff
@@ -1,30 +1,35 @@
 import {Page} from 'puppeteer';
+import BaseElement from './base-element';
 
 export default class SelectComponent {
 
-  constructor(private readonly page: Page, private readonly label?: string) {
+  constructor(private readonly page: Page, private readonly label?: string, private readonly nodeLevel?: number) {
     this.page = page;
     this.label = label || undefined;
+    this.nodeLevel = nodeLevel || 1;
   }
 
-  async select(textValue: string) {
+  async select(textValue: string): Promise<string> {
     await this.open(2); // with 2 retries
-    const selector = this.componentXpath() + `//li[@class='p-dropdown-item'][normalize-space(.)="${textValue}"]`;
+    const selector = this.dropdownXpath() + `//li[contains(normalize-space(text()), "${textValue}")]`;
     const selectValue = await this.page.waitForXPath(selector, { visible: true });
+    const selectElement = new BaseElement(this.page, selectValue);
+    const str = await selectElement.getTextContent();
```

Comment:
super nit: change variable name from str to textContent

Reply:
👍